packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -38,20 +38,6 @@ async function executeCompatibilityCheck(templateDir, yarnInstall) {
   })
   tui.startReactive(tuiContent)
 
-  if (!yarnInstall) {
-    tuiContent.update({
-      spinner: {
-        enabled: false,
-      },
-      content: `${RedwoodStyling.warning(
-        '⚠'
-      )} Skipped compatibility check because yarn install was skipped via command line flag`,
-    })
-    tui.stopReactive()
-
-    return
-  }
-
```

Comment:
I couldn't come up with a good reason why this compatibility check was dependent on yarn install. @Josh-Walker-GM was there one or was it just something we've inherited at this point?

Reply:
I don't know if that was the reason for it, but I do know I've used this to get around the node version check when I've wanted to run RW on a Node version that's too new.
For the new version check logic we have now that reason is not valid anymore.

Reply:
Yeah agreed with @Tobbe, I used it was an escape route when running above the node compatibility range. I can't think of a reason for it to stay either.